_states/ext_state_qvm.py

```diff
@@ -97,23 +97,24 @@
 def __virtual__():
     '''
     Only make these states available if a qvm provider has been detected.
-    '''
-    # XXX: TEMP
-    if not hasattr(qubes_utils, '__opts__'):
-        qubes_utils.__opts__ = __opts__  # NOQA
-    if not hasattr(qubes_utils, '__salt__'):
-        qubes_utils.__salt__ = __salt__  # NOQA
+    ''' # pylint: disable=multiple-statements
+    try: __opts__
+    except NameError: __opts__ = __opts__
+    try: __salt__
+    except NameError: __salt__ = __salt__
```

Comment:
This is change in behavior - previously it set `qubes_utils.__opts__`, now it set it in local namespace. Was that intentional?